src/RestaurantContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { loadRestaurant } from './actions';
+
+import { get } from './utils';
+
+export default function RestaurantContainer() {
+  const { id } = useParams();
+  const { name, address, menuItems } = useSelector(get('restaurant'));
+  const { loading } = useSelector((state) => ({
+    loading: state.loading,
+  }));
```

Comment:
```suggestion
  const { loading } = useSelector(get('loading'));
```
get 함수를 써서 가지고올수 없는 상황인가요?!

Reply:
loading 자체가 object가 아니라 true/false 값이어서.. `const loading = useSelector(get('loading'));`으로 수정했습니다. object destructuring이 익숙해질랑 말랑 하네요. ^^